src/components/Display.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Display extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const { result } = this.props;
+    return (
+      <div className="App">
```

Comment:
[Optional]

Let's use a functional component instead since `Display` does not need to be stateful